definitions/variable/industry/demand.yaml

```diff
@@ -0,0 +1,33 @@
+# material demand by commodity
+- Demand|Iron and Steel|Steel:
```

Comment:
Suggest to rename this to "Material Demand" or similar for consistency with "Agricultural Demand" and "Forestry Demand" in common-definitions.